app/controllers/provider/admin/backend_apis/metrics_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApis::MetricsController < Provider::Admin::BackendApis::BaseController
+  before_action :find_metric, except: %i[new create index]
+  before_action :build_metric, only: %i[new create]
+
+  # FIXME: This should be removed. It is still needed because either
```

Comment:
should we create a ticket? like the others todos that you commented?

Reply:
I think it's contained in https://issues.jboss.org/browse/THREESCALE-3089